CONTRIBUTORS.yaml

```diff
@@ -49,6 +49,8 @@ dannon:
 dpryan79:
     name: Devon Ryan
 
+dsobral:
```

Comment:
ping @dsobral: since you contributed to this repo we would like to add you to our CONTRIBUTORS hall of fame. However, I do not know your full name, let me know if you would like that to be added here :)

Reply:
@shiltemann `Daniel Sobral` from Portugal.

Reply:
thanks!